integration-cli/daemon/daemon.go

```diff
@@ -150,6 +148,21 @@ func (d *Daemon) WaitRun(contID string) error {
 	return WaitInspectWithArgs(d.dockerBinary, contID, "{{.State.Running}}", "true", 10*time.Second, args...)
 }
 
+// CmdRetryOutOfSequence tries the specified command against the current daemon for 10 times
+func (d *Daemon) CmdRetryOutOfSequence(args ...string) (string, error) {
```

Comment:
I think there’s a PR hat removes this, as it was no longer needed

Reply:
ohh interesting :stuck_out_tongue_closed_eyes:

Reply:
Let's remove this again in a follow up :+1: